### PR TITLE
Fix readme formatting, add note about Traits

### PR DESCRIPTION
**Pull request by M0rtyMerr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ expect(observable).first == 42
 
 Nice.
 
+This extension is also available for all observable convertible traits (e.g. `Single`, `Maybe`).
```
Comment (freak4pc) [inline README.md]:
```suggestion
This extension is also available for all Traits (e.g. `Single`, `Maybe`) and other types conforming to `ObservableConvertibleType`.
```

File: README.md [hunk 2]
```diff
@@ -35,7 +37,7 @@ expect(subject).events(scheduler: scheduler, disposeBag: disposeBag)
         Recorded.next(5, "Hello"),
         Recorded.next(10, "World"),
         Recorded.completed(100)
-        ]))
+     ]))
```
Comment (freak4pc) [inline README.md]: I think it's aligned to the `t` and not the `.`, right ? 
```suggestion
    ]))
```

File: README.md [hunk 3]
```diff
@@ -44,7 +46,7 @@ You may also verify specific error types:
 expect(imageSubject).events(scheduler: scheduler, disposeBag: disposeBag)
     .to(equal([
         Recorded.error(5, ImageError.invalidImage)
-        ]))
+     ]))
```
Comment (freak4pc) [inline README.md]: same

**Review by freak4pc — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (M0rtyMerr) [reply README.md]: Fixed, sure :)

File: README.md (hunk 1 above)
Comment (M0rtyMerr) [reply README.md]: Sure, thank!

File: README.md (hunk 3 above)
Comment (M0rtyMerr) [reply README.md]: Fixed, sure :)

File: README.md (hunk 2 above)
Comment (freak4pc) [inline README.md]: not sure if im imagining but it still looks unaligned

Comment (M0rtyMerr) [reply README.md]: Oh, now I get your idea, sorry :) 
Really strange case of formatting. I hope now it looks ok

**Review by ashfurrow — APPROVED** [review]
Looks good, thanks for addressing that feedback!

**Pull request merged**